src/styles/styles.scss

```diff
@@ -1 +1,7 @@
-@import 'colors';
+@import '_colors';
+@import '_extensions';
+@import '_variables';
+
+button {
```

Comment:
This is a weird rule. A button should not have a pointer cursor. Target specific button that should do this

Reply:
Agreed and fixed